.github/workflows/npm_package_pr_npm.yaml

```diff
@@ -33,7 +35,7 @@ concurrency:
 jobs:
   validate:
     name: Validate
-    runs-on: ${{ inputs.runs_on }}
+    runs-on: ${{ (inputs.fallback_runner && 'ubuntu-latest') || fromJson('["jupiterone-dev", "arm64"]') }}
```

Comment:
Isn't the fallback runner `ubuntu-latest` going to be `amd64`, which would still cause issues?

Reply:
Yes, it will. But we really don't want anyone using the fallback anyway. That's only really there if something is catastrophically wrong with the J1 runners and we have to get an emergency bundle out. In those cases we can touch the package-lock.json file and flip both the PR and the Deploy to the fallback. That will rebuild the cache for the fallback. We could make it so the cache is runner aware but it felt like more work than required at this stage.